helm-charts/common/whisper/templates/configmap.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "whisper.fullname" . }}-config
+  labels:
+    {{- include "whisper.labels" . | nindent 4 }}
+data:
+  EASYOCR_MODULE_PATH: "/tmp/.EasyOCR"
+  http_proxy: {{ .Values.global.http_proxy | quote }}
+  https_proxy: {{ .Values.global.https_proxy | quote }}
+  no_proxy: {{ .Values.global.no_proxy | quote }}
+  HF_HOME: "/tmp/.cache/huggingface"
+  HUGGINGFACE_HUB_CACHE: "/data"
```

Comment:
ASR_MODEL_PATH is the env to control ASR model https://github.com/opea-project/GenAIComps/blob/main/comps/asr/whisper/whisper_model.py#L27C50-L27C64

Reply:
Yes, I have included that env var in the values.yaml but comment out because not tested. Will do more test and enable it.